From 4bb211621c51f7cc58d833f0bbe47d887d51cc00 Mon Sep 17 00:00:00 2001
From: Tomas Zak <tomas.zak@turris.com>
Date: Thu, 20 Jun 2024 15:48:45 +0200
Subject: [PATCH] packages: python3: fix building on turris1x

Fix building python3 on turris1x. Due to different compile library for powerpc
python3 failed to build
---
 ...855-Refactor-platform-triplet-detection-code-GH-107221.patch | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lang/python/python3/patches/100-gh-95855-Refactor-platform-triplet-detection-code-GH-107221.patch b/lang/python/python3/patches/100-gh-95855-Refactor-platform-triplet-detection-code-GH-107221.patch
index 8f1ae5bdc..0a6aa25fc 100644
--- a/lang/python/python3/patches/100-gh-95855-Refactor-platform-triplet-detection-code-GH-107221.patch
+++ b/lang/python/python3/patches/100-gh-95855-Refactor-platform-triplet-detection-code-GH-107221.patch
@@ -151,7 +151,7 @@ Signed-off-by: Jeffery To <jeffery.to@gmail.com>
 +#    endif
 +#   endif
 +#   if defined(_SOFT_FLOAT) || defined(__NO_FPRS__)
-+#    define LIBC_PPC muslsf
++#    define LIBC_PPC muslspe
 +#   else
 +#    define LIBC_PPC musl
 +#   endif
-- 
2.45.2

